src/modules/pos/validator/components/SentStakes/SentStakes.js

```diff
@@ -31,12 +32,26 @@ function useStakerAddress(searchParam) {
 function ClaimRewardsDialogButton({ address }) {
   const { t } = useTranslation();
   const { data: rewardsClaimable } = useRewardsClaimable({ config: { params: { address } } });
+  const { hasSufficientBalanceForFee, feeToken } = useValidateFeeBalance();
+
+  const getInSuffienctBalanceMessage = () => {
+    if (!hasSufficientBalanceForFee) {
```

Comment:
Why doesnt this require `(!hasAvailableTokenBalance)` as well?

Reply:
For this, they don't require it because the only debit value incurred by the customer when claiming a reward or unlocking a token is the fee not the token balance of the user